LogMonitor/src/LogMonitor/Utility.cpp

```diff
@@ -324,4 +325,39 @@ bool Utility::CompareWStrings(wstring stringA, wstring stringB)
                 return towupper(l1) == towupper(r1);
             }
         );
+}
+
+std::wstring Utility::FormatEventLineLog(_In_ std::wstring customLogFormat, _In_ void* pLogEntry, _In_ std::wstring sourceType)
+{
+    size_t i = 0, j = 1;
+    while (i < customLogFormat.size()) {

```

Comment:
I wanted to suggest that we could use `<regex>` instead of the custom approach but looks like this approach will be shorter and perhaps more efficient 👍

Reply:
Thank you.